.github/workflows/php.yml

```
@@ -0,0 +1,61 @@
+name: PHP Composer
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: ['7.2', '7.3', '7.4']
```

Would be nice to spread across multiple lines so it can be more easily edited in future, IMHO.

```suggestion
        php-version:
          - '7.2',
          - '7.3'
          - '7.4'
```